### PR TITLE
Changed tags to Pending, Confirm, Not Attending

### LINKED ISSUE #538: Change to Pending, Confirmed, Not Attending
image

- More consistent with the other pages

### PR DESCRIPTION
**Pull request by AgentEinstein**
image

### CODE REVIEW

**Comment by shahdivyank** [comment]
Attach Issue

**Comment by AgentEinstein** [comment]
done

File: src/data/Tags.js [hunk 1]
```diff
@@ -56,9 +56,6 @@ export const teamTags = [
   {
     text: "disqualify",
   },
-  {
```
Comment (shahdivyank) [inline src/data/Tags.js]: why is this here?

Comment (AgentEinstein) [reply src/data/Tags.js]: Ah, I think I accidentally removed one of the tags from teamTags. I'll go fix it.

**Comment by AgentEinstein** [comment]
Fixed teamTags issue

**Review by shahdivyank — APPROVED** [review]

**Review by menthy-wu — APPROVED** [review]
